src/cephadm/cephadm

```diff
@@ -4903,7 +4928,11 @@ def _get_parser():
     parser_bootstrap.add_argument(
         '--initial-dashboard-password',
         help='Initial password for the initial dashboard user')
-
+    parser_bootstrap.add_argument(
+        '--ssl-dashboard-port',
+        type=int,
+        default = 8443,
+        help='Port number used to connect with dashboard using SSL')
```

Comment:
I'm getting increasingly worried about adding too many config entries as parameters to `bootstrap`. 

I'd like to have a general parameter `--set-config=what/key=value` or something. So this would end up as `--set-config=mgr/mgr/dashboard/ssl_server_port=8443` or something similar (maybe more readable). 

What's next? `--dashboard-server-port`? `--dashboard-url-prefix`?  

wdyt? Add a generic way or add this one as an explicit parameter?

(In any case you need to add the cephadm's man page)

Reply:
I think that it is sensible to provide this parameter as explicit. Because probably the main workflow will be to create a bootstrap cluster and start to "complete" the cluster using the dashboard. So having the dashboard using the most probable and definitive configuration (SSL using the port desired) from the very beginning seems to be sensible.

I do not thing that using a general parameter for manage other config settings will help us. Because probably we will need to introduce logic to manage/process properly these parameters inside cephadm. 

I think we should limit the parameters in the bootstrap command to only the minimum to create the bootstrap cluster with the "user side" config, the port used in the dashboard is a very clear example. 
IMHO, We have several parameters in the bootstrap command that probably we can move to a "cephadm config file" in the case that the user need to use a different setting from the default or we need to do a "very special" configuration.
Probably this must be object of a separate PR for a separate new FR "Simplify bootstrap command"